frontend/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": false,
+  "printWidth": 80
```

Comment:
Can we remove this please? VSCode can deal with line length so this does not seem helpful and potentially annoying.